project/src/main/java/com/google/sps/servlets/EditEventServlet.java

```diff
@@ -88,11 +95,34 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       event.setProperty("attendance", attendance);
       event.setProperty("description", description);
       event.setProperty("dateTimestamp", dateTimestamp);
+      if (imageKey != null) {
+        event.setProperty("imageKey", imageKey);
```

Comment:
what happen if directly set `event.setProperty("imageKey", imageKey);` when `imagekey` is `null`

Reply:
Before I put in this if statement I was getting a NullPointer error when trying to serve the blob, however I am still not sure why this fixes it.

Reply:
Maybe the entity doesn't accept `null` as value. Lets keep the `if` statement here.